plugin/audit_log/audit_log.c

```diff
@@ -1471,9 +1473,13 @@ static MYSQL_SYSVAR_STR(include_commands, audit_log_include_commands,
        audit_log_include_commands_update, NULL);
 
 static MYSQL_THDVAR_STR(local,
-                        PLUGIN_VAR_READONLY | PLUGIN_VAR_MEMALLOC | \
-                        PLUGIN_VAR_NOSYSVAR | PLUGIN_VAR_NOCMDOPT,
-                        "Local store.", NULL, NULL, "");
+       PLUGIN_VAR_READONLY | PLUGIN_VAR_MEMALLOC | \
+       PLUGIN_VAR_NOSYSVAR | PLUGIN_VAR_NOCMDOPT,
+       "Local store.", NULL, NULL, "");
+
+static MYSQL_THDVAR_ULONG(local_ptr,
```

Comment:
since static_assert is C++11, while audit_log.c is C99, I added DBUG_ASSERT. We would probably want to convert this plugin to C++, but it is out of scope of this fix.